libpod/kube.go

```diff
@@ -303,12 +303,23 @@ func containerToV1Container(c *Container) (v1.Container, []v1.Volume, error) {
 	// This should not be applicable
 	//container.EnvFromSource =
 	kubeContainer.Env = envVariables
-	// TODO enable resources when we can support naming conventions
-	//container.Resources
 	kubeContainer.SecurityContext = kubeSec
 	kubeContainer.StdinOnce = false
 	kubeContainer.TTY = c.config.Spec.Process.Terminal
 
+	if c.config.Spec.Linux != nil &&
```

Comment:
Can you add a TODO for adding CPU limiting support?

Reply:
Added. Thanks @mheon.